pkg/webui/console/containers/users-table/index.js

```diff
@@ -133,14 +150,15 @@ export default class UsersTable extends Component {
     return (
       <FetchTable
         entity="users"
-        headers={headers}
+        headers={this.headers}
         addMessage={sharedMessages.userAdd}
         tableTitle={<Message content={sharedMessages.users} />}
         getItemsAction={this.getUsersList}
         searchItemsAction={this.getUsersList}
         baseDataSelector={this.baseDataSelector}
         pageSize={pageSize}
         searchable
+        {...this.props}
```

Comment:
I think it's better to avoid using spread operators for props, especially if it initially works and we defined all props that are supplied to the component. Is this right @kschiffer ?

Reply:
```suggestion
```

It depends. @asmulko, you likely added this because we do this in other table components as well. I was more lax regarding this until experiencing some issues. When spreading props, it's important to make sure that no unrelated props are being passed down, since that can lead to nasty bugs. To easily avoid that, it's usually better to be explicit about what props to pass (as in not using spread). In this particular case, there's also no use in passing the props.